packages/Select/src/select.scss

```diff
@@ -0,0 +1,103 @@
+  --ids-select-font-weight: #{tokens.$font-weight-regular};
+  --ids-select-line-height: #{tokens.$space-4};
+  --ids-select-height: #{tokens.$space-6};
+  --ids-select-margin: #{tokens.$space-1} #{tokens.$space-0};
+  --ids-select-color: #{tokens.$grey-800};
+  --ids-select-chevron: #{tokens.$grey-800};
+  --ids-select-background: #{tokens.$samoyed};
+  --ids-select-border-color: #{tokens.$grey-300};
+  --ids-select-border-radius: #{tokens.$border-radius-sm};
+  --ids-select-number-of-option-to-display: 4;
+
+  /* Compact */
+  --ids-select-font-size-compact: #{tokens.$font-size-3};
+  --ids-select-line-height-compact: #{tokens.$line-height-xxl};
+  --ids-select-height-compact: #{tokens.$space-5};
+
+  /* Active */
+  --ids-select-border-color-active: #{tokens.$electric-blue-500};
+
+  /* Disabled */
+  --ids-select-color-disabled: #{tokens.$grey-500};
+  --ids-select-background-disabled: #{tokens.$grey-200};
+  --ids-select-border-color-disabled: #{tokens.$grey-200};
+
+  /* Error */
+  --ids-select-border-color-error: #{tokens.$coral-500};
+}
+
+.ids-select {
+  font-size: var(--ids-select-font-size);
+  font-family: var(--ids-select-font-family);
+  font-weight: var(--ids-select-font-weight);
+  line-height: var(--ids-select-line-height);
+  height: var(--ids-select-height);
+  width: 100%;
+  padding: 0 var(--ids-select-padding);
+  position: relative;
+  background-color: var(--ids-select-background);
+  border: 1px solid var(--ids-select-border-color);
+  border-radius: var(--ids-select-border-radius);
+  box-sizing: border-box;
+  appearance: textfield;
+  transition: border-color 200ms ease 0s;
+  margin: var(--ids-select-margin);
+  cursor: pointer;
+  user-select: none;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+
+  &:focus,
+  &.focus {
+    outline: 0;
+    box-shadow: none;
+    border-color: var(--ids-select-border-color-active);
+  }
+
+  .ids-dropdown {
+    position: absolute;
+    top: calc(var(--ids-select-height) - 1px);
+    left: -1px;
+    max-height: calc(var(--ids-select-height) * var(--ids-select-number-of-option-to-display));
+    overflow: auto;
+    display: flex;
+    flex-flow: column nowrap;
+  }
+
+  &.ids-select--compact {
+    font-size: var(--ids-select-font-size-compact);
+    line-height: var(--ids-select-line-height-compact);
+    height: var(--ids-select-height-compact);
+
+    .ids-dropdown {
+      top: calc(var(--ids-select-height-compact) - 1px);
+      max-height: calc(var(--ids-select-height-compact) * var(--ids-select-number-of-option-to-display));
+    }
+  }
+
+  &.ids-select--active,
+  &.active {
+    border-color: var(--ids-select-border-color-active);
+  }
+
+  &.ids-select--disabled {
+    color: var(--ids-select-color-disabled);
+    background-color: var(--ids-select-background-disabled);
+    border-color: var(--ids-select-border-color-disabled);
+    cursor: default;
```

Comment:
for the disabled option, you put `cursor: not-allowed;` 
How comes you didn't do it for the actual disabled select?

Reply:
When writing the `Select` classes I just forgot that we normally set `cursor: not-allowed;` do disabled components.

I will fix that oversight ✏